flink-end-to-end-tests/test-scripts/common.sh

```diff
@@ -316,7 +316,7 @@ function check_logs_for_errors {
       | grep -v "java.lang.NoClassDefFoundError: org/apache/hadoop/yarn/exceptions/YarnException" \
       | grep -v "java.lang.NoClassDefFoundError: org/apache/hadoop/conf/Configuration" \
       | grep -v "org.apache.flink.fs.shaded.hadoop3.org.apache.commons.beanutils.FluentPropertyBeanIntrospector  - Error when creating PropertyDescriptor for public final void org.apache.flink.fs.shaded.hadoop3.org.apache.commons.configuration2.AbstractConfiguration.setProperty(java.lang.String,java.lang.Object)! Ignoring this property." \
-      | grep -ic "error")
+      | grep -c "ERROR")
```

Comment:
As I have written in the ticket:

> This global `check_logs_for_errors ` is broken design. We might want to have some limited global list of exclusions, but adding more entries to ever growing list is unmaintainable. I think we should filter out this one specific warning in kafka tests and do not mess with changing global rules for all of the tests.

Reply:
Agree. I thought that you want to filter the warning log globally. Sorry, I misunderstood what you mean.

Reply:
Initially I though about ignoring all warnings, since it seemed to me as a better solution compared to changing the behaviour of case sensitiveness globally. But Then I realised that there are already way too many exclusions hardcoded in this file.

Reply:
@dawidwys I think you are also working on improving the e2e tests here. What is your opinion?

Reply:
First of all I agree this is a bit of a broken design, but the current version is that we do not want to introduce a major rework in the bash scripts. The idea right now is that if test requires some custom validation it should disable the default checking and do it in the test itself, so that this decision is a conscious one.

Having that in mind:
- I would definitely leave the case sensitivity as it is
- If you don't want to extend the global list, in other words if you don't want to rely on the default behavior I would disable the log checking via #6959 and add a custom check in the test itself that verifies logs with your extended whitelist (preferably in a trap invoked on exit).

Reply:
Thanks for your discussion, will try to change the implementation tomorrow.